lib/ts/recipe/passwordless/components/features/linkClickedScreen/index.tsx

```diff
@@ -70,9 +70,15 @@ class LinkClickedScreen extends PureComponent<PropType, unknown> {
                 });
             }
             if (response.status === "OK") {
+                const loginAttemptInfo = await this.props.recipe.recipeImpl.getLoginAttemptInfo();
                 await this.props.recipe.recipeImpl.clearLoginAttemptInfo();
+                const pathFromUrl = getRedirectToPathFromURL();
```

Comment:
It might be defined - devs might want to set this information on the backend while generating the link. I'm unsure if this is a use-case, but I wanted to keep the option open.

Reply:
For now, this is not a use case. We can add this as a use case to third party and to passwordless in another PR. For now, let's keep it consistent.